recipes/bottle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "bottle" %}
+{% set version = "0.12.9" %}
+{% set sha256 = "45285ad084ca054d821ceef8bd95462efd38e13ecbef13a82c22b6472a6f4b2d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bottlepy/bottle/archive/{{ version  }}.tar.gz
```

Comment:
As for the other part of your question, we have these [guidelines]( https://github.com/conda-forge/conda-forge.github.io/blob/1c12a1d6f82064c43575997138c6990c031eb9cc/docs/guidelines.rst ). Though they don't mention this point. It came up in a meeting instead. Perhaps it should be added.